app/assets/javascripts/components/revisions/diff_viewer.jsx

```diff
@@ -156,11 +164,43 @@ const DiffViewer = React.createClass({
     const wikiDiffUrl = this.webDiffUrl();
 
     let diffComment;
-    // Only show the edit summary for a single revision; for cumulative diffs,
-    // the edit summary is for the *parent* of the first revision, so it's
-    // irrelevant and confusing to show that.
+    let revisionDateTime;
+    let firstRevTime;
+    let lastRevTime;
+    let days;
+    let hours;
+    let minutes;
+    let timeSpan = '';
+
+    // Edit summary for a single revision:
+    //  > Edit date and number of characters added
+    // Edit summary for range of revisions:
+    //  > Span of time for edits to article (from first applicable rev to last)
     if (!this.props.first_revision) {
-      diffComment = <p className="diff-comment">{this.state.comment}</p>;
+      revisionDateTime = moment(this.props.revision.date).format('YYYY/MM/DD h:mm a');
+      diffComment = <p className="diff-comment">{this.state.comment}&nbsp;
+                       (Edited on {revisionDateTime};&nbsp;
+                       {this.props.revision.characters} chars changed)</p>;
+    } else {
+      firstRevTime = moment(this.state.firstRevDateTime);
+      lastRevTime = moment(this.state.lastRevDateTime);
+      days = lastRevTime.diff(firstRevTime, 'days');
+      hours = lastRevTime.diff(firstRevTime, 'hours') - days * 24;
+      minutes = lastRevTime.diff(firstRevTime, 'minutes') - (days * 24 + hours) * 60;
+      if (days !== 0) {
+        timeSpan = `${days} day${days > 1 ? 's' : ''}, `;
+      }
+      if (hours !== 0) {
+        timeSpan += `${hours} hour${hours > 1 ? 's' : ''}, `;
+      }
+      if (minutes !== 0) {
+        timeSpan += `${minutes} minute${minutes > 1 ? 's' : ''}`;
+      }
+      if (!timeSpan) {
+        timeSpan = 'less than a minute';
```

Comment:
Kind of like that .... but I don't want to be limited to ```now``` as the reference time, since the span I was looking for was the duration between datetime1 and datetime2.  

What I needed was the JS equivalent to: http://api.rubyonrails.org/classes/ActionView/Helpers/DateHelper.html#method-i-distance_of_time_in_words

But, it is moot since we'll just use timestamps for the editing period end points.

Thanks for the explanation explanation of hot the i18n should be done.